spec/spec_helper.rb

```diff
@@ -1,10 +1,19 @@
 require "active_record"
 
+sqlite_file = "file::memory:?cache=shared"
+FileUtils.rm_f(sqlite_file)
+
 ActiveRecord::Base.establish_connection(
   :adapter => "sqlite3",
-  :database => "file::memory:?cache=shared"
```

Comment:
FYI this broke in a new version of sqlite 😞 
https://stackoverflow.com/questions/14205928/activerecord-in-memory-shared-cache